### PR TITLE
[FD-386] Requests caching

### PR DESCRIPTION
**Pull request by starkyru**

### CODE REVIEW

File: src/screens/Namespaces/components/NamespacesTableFilters/NamespacesTableFilters.tsx [hunk 1]
```diff
@@ -94,6 +95,8 @@ interface INamespacesTableHeaderProps {
   applyFilters: (fliters: INamespaceFilters) => void;
 }
 
+const FETCH_DELAY = 300;
```
Comment (BigBadAlien) [inline src/screens/Namespaces/components/NamespacesTableFilters/NamespacesTableFilters.tsx]: ms?

File: src/redux/actions/namespacesActions.ts [hunk 2]
```diff
@@ -41,6 +42,9 @@ const NamespacesActions = {
       url: '/namespace/list',
       method: 'GET',
     },
+    meta: {
+      cache: NAMESPACES_CACHE_TIME,
```
Comment (BigBadAlien) [inline src/redux/actions/namespacesActions.ts]: Is it middleware? Hot it works?

File: src/screens/Namespaces/components/NamespacesTableFilters/NamespacesTableFilters.tsx [hunk 3]
```diff
@@ -108,7 +111,11 @@ const NamespacesTableHeaderComponent: React.FunctionComponent<
   } = props;
 
   useEffect(() => {
-    fetchDatacenters();
+    useTimeout(() => {
+      if (!isRequestInProgress(datacentersFetchStatus)) {
+        fetchDatacenters();
+      }
+    }, FETCH_DELAY);
```
Comment (BigBadAlien) [inline src/screens/Namespaces/components/NamespacesTableFilters/NamespacesTableFilters.tsx]: What's sense to delay it?

File: src/redux/actions/namespacesActions.ts (hunk 2 above)
Comment (starkyru) [reply src/redux/actions/namespacesActions.ts]: https://www.npmjs.com/package/redux-saga-requests#cache-middleware-arrow_up

**Pull request merged**